src/components/Calls/ShowJoinCallButton.js

```diff
@@ -0,0 +1,43 @@
+import { h } from 'preact';
```

Comment:
We ususally don't include a verb within a component name. Perhaps we can simply name this component as `JoinCallButton`